.github/workflows/continuous-integration-cypress.yml

```diff
@@ -1,15 +1,19 @@
 name: Cypress tests
 
 on:
-  push:
+  workflow_run:
+    workflows: ["Deploy to environment (CIP)"]
     branches: [ main ]
+    types:
+      - completed
 
 env:
   NODE_VERSION: 18.x
 
 jobs:
   cypress-tests:
-    environment: staging
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
@jordan-beard this line ensures that only successful deployments will trigger the test job

Reply:
I don't think we would want to run the tests when we deploy to production.